ext/FillArraysPDMatsExt.jl

```diff
@@ -0,0 +1,18 @@
+module FillArraysPDMatsExt
+
+if isdefined(Base, :get_extension)
```

Comment:
This style seems a bit unusual to me. Would it not be possible to import the packages directly, or is the namespace specification necessary?

Reply:
The branch is necessary, Requires needs relative imports (as shown also in the docs: https://github.com/JuliaPackaging/Requires.jl and https://pkgdocs.julialang.org/v1/creating-packages/#Requires.jl).

Moreover, in an extension one should import only the package itself and its weak dependency, and even stdlibs should be loaded indirectly: https://github.com/JuliaStats/LogExpFunctions.jl/pull/63

Reply:
Oh, interesting, I had no idea about this transitive loading issue. Looks fine to me then